src/store/metadata/stub/content/editions/cancer-diagnosis-at-stage-1-and-2_2023-05.json

```diff
@@ -0,0 +1,116 @@
+{
+  "@context": "https://staging.idpd.uk/ns#",
+  "@id": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions",
+  "@type": "hydra:Collection",
+  "title": "Cancer diagnosis at stage 1 and 2",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions/2023-05",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2",
+      "identifier": "2023-05",
+      "title": "Cancer diagnosis at stage 1 and 2",
+      "summary": "New cases of cancer diagnosed at stages 1 and 2 as a percentage of all new cases of cancer diagnosed at any known stage, England, 2013 to 2020.",
+      "description": "This dataset shows new cases of cancer diagnosed at stages 1 and 2 as a percentage of all new cases of cancer diagnosed at any known stage, in England, for 2013 to 2020.",
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "contact_point": {
+        "name": "Office for health improvement and disparities",
+        "email": "mailto:dhsc.publicenquiries@dhsc.gov.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/health-and-social-care"
+      ],
+      "frequency": "annual",
+      "keywords": [
+        "tumor",
+        "subnational",
+        "local-authority",
+        "county",
+        "region"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-05-17T12:25:53Z",
+      "modified": "2023-08-17T11:55:00Z",
+      "spatial_resolution": [
+        "N09",
+        "S12",
+        "W06",
+        "E92",
+        "K02"
+      ],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": [
+        "P1M",
+        "P3M",
+        "P1Y"
+      ],
+      "temporal_coverage": {
+        "start": "2013-01-01T12:00:00Z",
+        "end": "2020-12-31T12:00:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions/2023-05/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions/2023-05/versions/1",
+          "issued": "2023-05-17T12:25:53Z"
+        }
+      ],
+      "next_release": "2024-05-01T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Year",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "Statistical Geography",
+            "description": "The statistical geography associated with the observation."
+          },
+          {
+            "name": "value",
```

Comment:
Just for consistency between other json metadata files change this to 'observation'.